### PR TITLE
feat: implement ruby cloud function to translate text

### PR DESCRIPTION
**Pull request by akashgp09**
Resolves: #97 

 Implemented ruby cloud function for translating text from one language to another using google translate.

 - [x]  Write code for the function.
 - [x]  Test the function using Appwrite.
 - [x]  Write a README.md explaining what function does, how to use and customize it.

### Sample Input:

Screenshot from 2021-10-08 23-17-01

### Sample Output :

Screenshot from 2021-10-08 23-16-43

### CODE REVIEW

File: ruby/translate-text/main.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+require 'json'
+
+# Get the text, sourceLanguage and destinationLanguage from Appwrite's environment variable
+payload = JSON.parse(ENV['APPWRITE_FUNCTION_DATA'])
+text=payload["text"]
+sourceLanguage=payload["sourceLanguage"]
+destinationLanguage=payload["destinationLanguage"]
+
+# Translate your text
+translation = GoogleTranslate.translate from: sourceLanguage, to: destinationLanguage, text: text
```
Comment (Meldiron) [inline ruby/translate-text/main.rb:12]: Can we include some way of authentification to GoogleTranslate? There might be some public API, but it surely is limited.  By authenticating, the limitation will be gone.

File: ruby/translate-text/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# 🌐 Translate text from one language to another
+
+A sample Ruby Cloud Function for translating text from one language to another using google translate.
+
```
Comment (Meldiron) [inline ruby/translate-text/README.md:4]: Can we please also include example input and output?

**Review by Meldiron — CHANGES_REQUESTED** [review]
Great PR! 🤯 Left some comments during the review, please check them out.

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**